streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -145,8 +142,8 @@ public void testRegexMatchesTopicsAWhenCreated() throws Exception {
         final KStream<String, String> pattern1Stream = builder.stream(Pattern.compile("TEST-TOPIC-\\d"));
 
         pattern1Stream.to(stringSerde, stringSerde, DEFAULT_OUTPUT_TOPIC);
-        final List<String> assignedTopics = new CopyOnWriteArrayList<>();
-        streams = new KafkaStreams(builder.build(), streamsConfig, new DefaultKafkaClientSupplier() {
+        final List<String> assignedTopics = new ArrayList<>();
+        final KafkaStreams streams = new KafkaStreams(builder.build(), streamsConfig, new DefaultKafkaClientSupplier() {
```

Comment:
`KafkaStreams` is AutoCloseable now so you can include its construction inside the `try` block. Ditto elsewhere.

Reply:
Thanks @guozhangwang this PR is for the `1.1` branch so `KafkaStreams` does not implement `AutoCloseable` in branches < `2.0`

Reply:
Ah got it, my bad :)